client/src/styles/AccountSettingsPage.scss

```diff
@@ -73,8 +73,10 @@
       display: flex;
       flex-direction: column;
       gap: 0.625;
+
       .send-new-link {
         justify-content: start;
+        align-items: center;
```

Comment:
I'm not seeing that this does anything - wasn't it only the `link-sent` change that was needed?

Reply:
yeah good catch!